seb_openedx/middleware.py

```diff
@@ -137,6 +139,17 @@ def handle_access_denied(self, request, view_func, view_args, view_kwargs, cours
         is_banned, new_ban = ban_user(user_name, course_key, request.user.username)
         is_courseware_view = bool(view_func.__name__ == get_courseware_index_view().__name__)
         context.update({"banned": is_banned, "is_new_ban": new_ban})
+
+        if not is_banned:
+            try:
+                usage_key_string = request.resolver_match.kwargs.get('usage_key_string')
+                url = reverse('jump_to', kwargs=dict(course_id=str(course_key), location=str(usage_key_string)))
```

Comment:
Could we use a dictionary directly instead of calling `dict()`?

Reply:
I took some code from the dashboard page as the inspiration. I have no particular inclination for or against dict

Reply:
The Pylint extension in VSCode suggested I to change it, here is the reference: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/use-dict-literal.html